uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/main/java/org/uberfire/ext/metadata/backend/infinispan/provider/InfinispanConfiguration.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.uberfire.ext.metadata.backend.infinispan.provider;
+
+import java.io.InputStream;
+import java.util.Scanner;
+
+import org.infinispan.commons.configuration.XMLStringConfiguration;
+
+public class InfinispanConfiguration {
+
+    public static final String INFINISPAN_INDEXED_TEMPLATE = "infinispan-indexed-template.xml";
+    public static final String INFINISPAN_NON_INDEXED_TEMPLATE = "infinispan-non-indexed-template.xml";
+
+    private final String template;
+    private final String nonIdexedTemplate;
+
+    public InfinispanConfiguration() {
```

Comment:
(optional nitpick) Initialize the fields directly and save 5 lines of code by removing constructor ;-)